cheeks.js

```diff
@@ -0,0 +1,48 @@
+const display = document.getElementById("display-info");
+const selector = document.querySelector("select");
+const button = document.querySelector("button");
+const reviewHolder = document.getElementById("reviews");
+const form = document.querySelector("form");
+const input = document.querySelector("input");
+const year = document.getElementById("release-year");
+const description = document.getElementById("description");
+const title = document.getElementById("title")
+
+const getTitles = async () => {
+  const allFilms = await axios.get("https://ghibliapi.herokuapp.com/films");
+  allFilms.data.forEach((film) => {
+    let option = document.createElement("option");
+    option.innerHTML = film.title;
```

Comment:
we could of used the `value` option property to store the id of the current movie
`option.value = film.id`